src/main/java/org/broadinstitute/dsde/consent/ontology/service/ElasticSearchAutocompleteAPI.java

```diff
@@ -84,7 +84,14 @@ public ElasticSearchAutocompleteAPI(ElasticSearchConfiguration configuration) {
                 logger.error("Unable to parse 'hits' from: " + stringResponse);
             }
         } catch (IOException e) {
-            logger.error("Unable to parse query response for " + query + "\n" + e.getMessage());
+            // Trap non-fatal "Connection reset..." errors
+            String message = "Unable to parse query response for " + query + "\n" + e.getMessage();
+            if (e.getMessage().contains("Connection reset by peer")) {
+                logger.warn(message);
+                return termList;
```

Comment:
Note to reviewer, I am on the fence about returning a (likely) empty term list here. This scenario comes up when ontology can't connect to ES for some spurious network related reason (correct me if I'm wrong). It isn't necessarily an error on either ES or ontology's side, it's just something that ontology can't recover from. Is it valuable to return that system-level information to the user?